Leuchtfeuer/mautic-DeleteContactHistory
# Delete Contact History by Leuchtfeuer

Description:
This PR itroduced new campaign actions, which can delete contact history:

It is possible to set which data You want to clear

Steps to test this PR:
Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))

1. Create campaign which based whether on segment or on form
2. Choose as an action "Delete Contacts History"
3. Add some contacts to this campaign
4. Run this campaign via `console m:c:r` => `console m:c:t`
